### DISCUSSION TITLE
A few tweaks in SumatraPDF-settings.txt formatting

### DISCUSSION THREAD

**Discussion by fabiorzfreitas** [sumatrapdfreader/sumatrapdf#4025] [Ideas]
I was making changes to my `SumatraPDF-settings.txt` file and I thought of a few improvements:

- Using `.toml` instead of `.txt`.
  - This would be useful for triggering text/code editors' language highlighting, among other functions (such as auto indentation, creating brackets in pairs etc.);
  - Editing a settings file is "advanced", so if Windows prompts "Open As", the user is expected to at least *try* to open it with Windows Notepad;

- Commenting code (exactly like the [documentation](https://www.sumatrapdfreader.org/settings/settings3-5-1.html)).
  - It would be easier to edit a commented file than going back and forth to the online docs;

- Storing `FileStates` and `SessionData` on a separate file.
  - These really clutter the settings file. When I first opened it, I thought I'd have to tackle a 700-liner beast!
  - As a bonus, this would also solve another Feature Idea: limiting the max number of entries in those sections. If they're on another file, they won't clutter anyone's settings.

That's it, I think these changes are feasible and would improve user experience!

**Comment by kjk** [comment]
Our format is not `.toml` so even if it would trigger better syntax highlighting, it might make people think that's toml and then they'll create bug reports if we don't parse toml syntax.

It's too late to split settings file.

There is a thousand of things more important than supporting comments.
